react/src/lib/components/DeckGLMap/DeckGLMap.jsx

```diff
@@ -193,6 +200,24 @@ DeckGLMap.propTypes = {
         pickDepth: PropTypes.number,
     }),
 
+    /**
+     * Parameters for the Distance Scale component
+     */
+    scale: PropTypes.shape({
+        /**
+         * Toggle component visibility.
+         */
+        visible: PropTypes.bool,
+        /**
+         * Increment value for the scale.
+         */
+        incrementValue: PropTypes.number,
+        /**
+         * width per unit value to calculate width in units.
```

Comment:
It would be good to specify that this value will be in pixels, so maybe something like:
"Scale bar width in pixels per unit value"